### PR TITLE
python code and its related tests for tensorflow DNNRegressionModel

### LINKED ISSUE #433: docs: model: tensorflow: Include examples of how to use from Python API
The model plugins page has command line examples but needs examples from Python as well.

Copy the format of the `examples/quickstart_filenames.py` file and add a `.. code-block:: python` section for each model, keeping the Python example consistent with the command line example for each model. Let's add the python example after the CLI example for each.

For tensorflow, the docstrings of the `Model`s themselves will need to be updated.

Here's a good example from when scikit was updated: https://github.com/intel/dffml/pull/407/commits/a407dc362e4250d143b256aae588af791d90ef08
Also reference this commit https://github.com/intel/dffml/commit/73715ae92446169b22fed2ec38ea2de43cfdad01 to see how we want to make the examples tested.

### PR DESCRIPTION
**Pull request by sk-ip**
modify the `tfdnnr` code examples and also added python code for the same. the tests will be added soon.
references #433 closes #433

### CODE REVIEW

**Comment by sk-ip** [comment]
@pdxjohnny , please review this pr, due to some reason the docs are failing and i couldn't find the reason.

File: docs/plugins/dffml_model.rst [hunk 1]
```diff
@@ -154,77 +154,55 @@ Generating train and test data
...
+        },
+        "last_updated": "2019-10-24T15:26:41Z",
+        "prediction": {
+            "TARGET" : {
+                "confidence": NaN,
+                "value": 1.1983429193496704
+            }
+        },
+        "key": 0
+      }
```
Comment (pdxjohnny) [inline docs/plugins/dffml_model.rst]: It looks like some of this JSON is indented with four spaces and some is indented with two. Can we make it all four?

**Comment by pdxjohnny** [comment]
@sk-ip Sorry I took so long to get back to you. Things have been very hectic!

**Comment by sk-ip** [comment]
@pdxjohnny , thanks for reviewing, i merged the master into this branch and also resolved the indentation issue. but when i run `./scripts/docs.sh` all the changes made to `dffml/docs/plugins/dffml_model.rst` in this pr are getting reverted.

**Comment by sk-ip** [comment]
Hello @pdxjohnny , i was able to solve the above mentioned problem, thanks to @0dust .

**Pull request merged**